### ISSUE TITLE
[Tooltip] Native "Reference element" label appearing upon hover in Addition to Tooltip

### ISSUE THREAD

**Issue by SkyeSeitz** [Esri/calcite-components#6211]
### Actual Behavior

The title field on the reference element, `calcite-button`, is populated with `"Reference element"` 
<img width="174" alt="image" src="https://user-images.githubusercontent.com/29716057/210104504-ef7f5595-55cb-4a21-b67c-cf9789a01400.png">

causing the native label to appear on reference element upon hover, as seen below.

Windows:
<img width="547" alt="image" src="https://user-images.githubusercontent.com/29716057/210104213-b8965bd4-96fc-4b64-99e7-6c04434e65e3.png">

Mac:
<img width="340" alt="image" src="https://user-images.githubusercontent.com/29716057/210104206-61cee621-0946-4177-aec1-6fdd265e0ae3.png">

### Expected Behavior

Do we need this that "title" field populated? 

cc @geospatialem can you confirm if it is required for a11y?

### Reproduction Sample

https://esri.github.io/calcite-components/?path=/story/components-tooltip--simple

### Reproduction Steps

1. Open the Storybook Tooltip sample
2. Hover on the `calcite-button` reference element
3. Wait for `calcite-tooltip` and native label to appear

### Reproduction Version

v1.0.0-beta.99

### Esri team

Calcite (design)

**Comment by driskull** [comment]
It seems like the storybook just needs to be updated to remove the title attribute on the button.

**Comment by geospatialem** [comment]
> It seems like the storybook just needs to be updated to remove the title attribute on the button.

Agree, there also seems to be a missing `span` for AT support. Will have a PR ready for a doc fix shortly.

**Comment by geospatialem** [comment]
Verified on [Storybook](https://esri.github.io/calcite-components/?path=/story/components-tooltip--simple).
